Snoop/WindowInfo.cs

```diff
@@ -148,6 +148,38 @@ public string Description
             }
         }
 
+        #region UI Binding sources
+
+        public string WindowTitle
```

Comment:
This new property should also be used in `Description`. Otherwise we would end up with two code paths extracting the window title.